actix-session/src/storage/redis_rs.rs

```diff
@@ -273,12 +289,11 @@ impl RedisSessionStore {
 
 #[cfg(test)]
 mod test {
-    use std::collections::HashMap;
-
-    use redis::AsyncCommands;
-
     use super::*;
     use crate::test_helpers::acceptance_test_suite;
+    use actix_web::cookie::time;
+    use redis::AsyncCommands;
+    use std::collections::HashMap;
```

Comment:
do you have rustfmt re-ordering of imports overriden :P

Reply:
I am running with the default config of rustfmt, nothing custom.

Reply:
On my machine, neither stable nor nightly re-order the imports automatically like this.

Reply:
Separating empty lines are needed to keep groups of imports.

Reply:
Oh, that might be it 😅

Reply:
Opened #237 to sort this with automation in the future.